Improve auth doc

ykdojo opened the pull request:
## What kind of change does this PR introduce?

Adding some description about how to change auth settings on our documentation.

ykdojo committed:
Update auth-signup.mdx

ykdojo committed:
Update auth-signin.mdx

ykdojo committed:
Update auth-signin.mdx

ykdojo committed:
Update auth.mdx

ykdojo committed:
typo: login -> log in

kiwicopple commented:
Hey @ykdojo - this is something I probably should have explained. All the client docs are autogenerated (enriched with our Typescript definitions).

So these pages are generated from `/web/spec/supabase.yml`. Could you please add the changes there too?

ykdojo committed:
vue3-ts-todo: make user sessions persistent upon refresh

ykdojo committed:
Revert "vue3-ts-todo: make user sessions persistent upon refresh"

This reverts commit 9f66c1535d288a42b27ab09eca5feb4007e5f29a.

ykdojo committed:
Re-generated a few docs with RefGenerator

web/docs/client/filter.mdx
web/docs/client/insert.mdx
web/docs/client/order.mdx

ykdojo commented:
Hey @kiwicopple, I just updated supabase.yml and re-generated the docs. It generated a few new things, but I think it's mostly just changing the order of the params.

Also, it tried to edit the following files, but I didn't include them in this PR to make it easier to read. I think I'll make a separate PR for them after this one.

```
web/docs/client/filter.mdx
web/docs/client/insert.mdx
web/docs/client/order.mdx
```
